templates/course-results/lessons.php

```diff
@@ -115,16 +115,12 @@
 		if ( 0 < count( $lessons ) ) :
 			?>
 
-			<h3>
-
-				<?php
-				// lesson title will already appear above
-				if ( $course_has_lessons_in_modules ) {
-					esc_html_e( 'Other Lessons', 'sensei-lms' );
-				}
+			<?php
+			// lesson title will already appear above
+			if ( $course_has_lessons_in_modules ) :
 				?>
-
-			</h3>
+				<h3><?php esc_html_e( 'Other Lessons', 'sensei-lms' ); ?></h3>
+			<?php endif; // $course_has_lessons_in_modules ?>
```

Comment:
Can this comment be removed? We're trying to move away from doing this.

Reply:
Good to know.